test/python/test_rsqrtgrad.py

```diff
@@ -0,0 +1,61 @@
+# ==============================================================================
+#  Copyright 2018-2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+"""nGraph TensorFlow bridge ReluGrad operation test
+
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pytest
+
+import tensorflow as tf
+from tensorflow.python.framework import constant_op
+from tensorflow.python.ops.gen_math_ops import rsqrt_grad
+
+from common import NgraphTest
+import numpy as np
+
+
+class TestRsqrtGrad(NgraphTest):
+
+    def test_rsqrtgrad_2d(self):
```

Comment:
Please use `parameterize` to avoid 2 separate tests. See https://github.com/tensorflow/ngraph-bridge/blob/master/test/python/test_prod.py for an example